benchopt/runner.py

```diff
@@ -236,6 +236,7 @@ def __init__(self, objective, max_iter, deadline, meta):
         self.it = 0
         self.time_iter = 0.
         self.next_stopval = 0
+        self.prev_objective = 1e10
```

Comment:
```suggestion
        self.prev_objective = np.inf
```
?

Reply:
I put `math.inf` as numpy is not imported in the runner